### PR TITLE
Remove the line breaks

### PR DESCRIPTION
**Pull request by NelHim**
Worked on reviews in managed teams and qualitat

### CODE REVIEW

File: src/sections/qualitat.html [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-                    dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est L vero eos et
-                    accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata
-                    sanctus est L
-                </span>
-            </p>
-        </div>
+<div class="relative z-0 px-4 -top-44 pt-52 lg:pt-52 pb-48 md:px-10 xl:px-36 ojemba-oblique2 2xl:grid justify-center">
+    <div> 
+        <p class="text-white md:w-1/2">
+            <span class="font-railwayBold text-xl leading-3 ">Qualität steht bei uns an erster Stelle. Mit unserem
```
Comment (ishimwe07) [inline src/sections/qualitat.html]: You can remove that space at the end of this class, I think there is no need for it.

File: src/sections/qualitat.html [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+                <span class="text-green">Trainingsprogramm</span> direkt Vorort werden die  juniorigen Mitarbeitenden auf das projektgeschäft <br> vorbereitet.</span> 
+            <br> <br>
```
Comment (ishimwe07) [inline src/sections/qualitat.html]: You can consider removing these line breaks (<br>) as it was highlighted in the code reviews from coaches.

File: src/sections/qualitat.html [hunk 3]
```diff
@@ -35,7 +34,7 @@
                 </section>
             </div>
         </div>
-    
+
         <h4 class="pt-12 text-base flex text-white">
             <section class="md:block">
                 <span class="relative z-10">Mehr dazu von Matthias Moosburger, Senior Lead IT Architect bei MaibornWolff,<br> der das Ausbildungsprogramm maßgeblich entwickelt hat:</span>
```
Comment (ishimwe07) [inline src/sections/qualitat.html]: You can also consider removing that <br> for this paragraph.

**Review by ishimwe07 — CHANGES_REQUESTED** [review]
Looks good, but I highlighted some lines that you can consider to work on, to make it much better.

**Review by NoellaPromise — APPROVED** [review]

**Review by ishimwe07 — APPROVED** [review]
LGTM

**Pull request merged**